lidar3d:

  downsampling:
    row_step: 1 # = 1080 / 64
    col_step: 1 # = 720 / 64

  clip:
    range:
      use: false
      min: 0.5
      max: 8.0